client/src/components/InventoryView.js

```diff
@@ -27,12 +27,29 @@ class InventoryView extends React.Component {
           this.state.isMobile ? (
             <p className="text-danger">Replace me with a mobile component</p>
           ) : (
-            <InventoryTable />
+            <InventoryTable table={[
+              {
+                id: "01",
+                name: "Raspberry Pi",
+                description: "Mini computer",
+                number: 2,
```

Comment:
We should call this `numberAvailable` and then remove the `available` property. Conditionally render based on the value of `numberAvailable` (i.e. 0 is not available and everything greater than 0 is available). I'll incorporate this into the API 👍

Reply:
Per your comment on slack better name for this would be `quantity`